enterprise/frontend/src/metabase-enterprise/feature_level_permissions/permission-management/download-permission.ts

```diff
@@ -21,7 +21,19 @@ import {
 import { getGroupFocusPermissionsUrl } from "metabase/admin/permissions/utils/urls";
 import { PLUGIN_ADVANCED_PERMISSIONS } from "metabase/plugins";
 
-export const DOWNLOAD_PERMISSION_REQUIRES_DATA_ACCESS = t`Download results access requires full data access.`;
+export const UNABLE_TO_DOWNLOAD_RESULTS = t`Users with Block data access can't download results`;
```

Comment:
I'm not sure this can be backported to the current release because it's adding a new translation string. cc @flamber @mazameli @jeff-bruemmer

Reply:
Maybe the solution is to remove the tooltip all together for this release (as @jeff-bruemmer suggested on Slack), and follow up with the newer version of the tooltip with this string, but point it to the v44.

Reply:
You can backport this change. I believe @mazameli  said in the #releases channel that he'd do a round of translations for the next point release.

```suggestion
export const UNABLE_TO_DOWNLOAD_RESULTS = t`Groups with Block data access can't download results`;
```

I'm suggesting we change "Users" to "Groups", as we assign permissions to groups, not people.

Thanks for catching this, and creating the fix, @nemanjaglumac!

Reply:
@jeff-bruemmer updated, thanks for your suggestion!